index.html

```diff
@@ -84,19 +83,19 @@ <h3 class="text-center">Gorilla Used at
                 <div class="col-md-12 text-center"><img src="assets/img/gorilla_method.png" 
                     style="width: 80%;margin-bottom: 8px;" alt="Gorilla LLM logo">
                 </div>
-                <em>Gorilla is a LLM that can provide appropriate API calls. 
+                <p class="col-md-12" style="color: black;margin-bottom: 2px; margin-left: 10px; margin-right: 10px;">Gorilla is a LLM that can provide appropriate API calls. 
```

Comment:
same here, why not just define a css rule for the `col-md-12` class in `main_page.css`? It has been repeated multiple times.